### PR TITLE
feat(repl): add global clear() function

### PR DESCRIPTION
**Pull request by cjihrig**
This commit adds a `clear()` function in the REPL which works similar to `console.clear()`.

Fixes: https://github.com/denoland/deno/issues/14226

### CODE REVIEW

**Review by dsherret — APPROVED** [review]
LGTM!

As mentioned in the issue, I think this makes sense to add because both Firefox and Chrome have this in their REPL.

**Review by bartlomieju — APPROVED** [review]
LGTM too

File: cli/tests/integration/repl_tests.rs [hunk 1]
```diff
@@ -776,6 +776,6 @@ fn pty_clear_function() {
     assert!(output.contains("[1;1H"));
     assert!(output.contains("undefined"));
     assert!(output.contains("const clear = 1 + 2;"));
-    assert!(output.contains("3"));
+    assert!(output.contains('3'));
```
Comment (lucacasonato) [inline cli/tests/integration/repl_tests.rs:779]: Wow, clippy is a little aggressive today... :D

**Review by lucacasonato — APPROVED** [review]
LGTM!

**Pull request merged**